challenges/objects-arrays.js

```diff
@@ -44,8 +69,17 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 
 /* Request 1: Create a new array called universities that contains all the univeristies in the graduates array.  
 
+
 Once you have the new array created, sort the universities alphabetically and log the result. */
+
+let index = graduates.length - 1 ;
 const universities = [];
+for (let i = 0 ; i <= index; i++) {
+  universities.push(graduates[i].university);
+  universities.sort();
```

Comment:
to save on complexity you should use .sort() outside of the for loop. It only needs to be done once.